package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@checkdigit/github-actions",
+  "version": "1.0.0",
+  "description": " Provides supporting operations for github action builds.",
+  "typings": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "prettier": "@checkdigit/prettier-config",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/checkdigit/github-actions.git"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "author": "Check Digit, LLC",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/checkdigit/github-actions/issues"
+  },
+  "homepage": "https://github.com/checkdigit/github-actions#readme",
+  "bin": {
+    "action": "bin/action"
+  },
+  "dependencies": {
+    "@octokit/rest": "^19.0.4",
+    "debug": "^4.3.4"
+  },
+  "devDependencies": {
+    "@checkdigit/eslint-config": "^7.2.2",
+    "@checkdigit/jest-config": "^1.0.1",
+    "@checkdigit/prettier-config": "^3.0.0",
+    "@checkdigit/typescript-config": "^3.1.0",
+    "@types/debug": "^4.1.7",
+    "@types/uuid": "^8.3.4",
+    "nock": "^13.2.9",
+    "rimraf": "^3.0.2",
+    "undici": "^5.10.0",
+    "uuid": "^9.0.0"
+  },
+  "scripts": {
+    "dist": "rimraf dist && tsc --outDir dist && rimraf dist/*.spec.*",
+    "prepublishOnly": "npm run dist && rimraf .github src test typings .eslintrc .gitignore tsconfig.json",
```

Comment:
instead of defining what we don't want to publish, I think going forward we should just specify what files *should* be published using the files property (https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files)

Reply:
that is a much better solution